mediapipe/pose_tracking_dll/pose_tracking.cpp

```diff
@@ -0,0 +1,179 @@
+#include <cstdlib>
+#include <string>
+
+#include "pose_tracking.h"
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "mediapipe/framework/formats/landmark.pb.h"
+#include "mediapipe/framework/calculator_framework.h"
+#include "mediapipe/framework/formats/image_frame.h"
+#include "mediapipe/framework/formats/image_frame_opencv.h"
+#include "mediapipe/framework/port/file_helpers.h"
+#include "mediapipe/framework/port/opencv_highgui_inc.h"
+#include "mediapipe/framework/port/opencv_imgproc_inc.h"
+#include "mediapipe/framework/port/opencv_video_inc.h"
+#include "mediapipe/framework/port/parse_text_proto.h"
+#include "mediapipe/framework/port/status.h"
+
+class PoseTrackingImpl {
+public:
+	PoseTrackingImpl(const std::string& calculatorGraphConfigFile) {
+		auto status = initialize(calculatorGraphConfigFile);
+		if (!status.ok()) {
+			LOG(WARNING) << "Warning: " << status;
+		}
+	}
+
+	absl::Status initialize(const std::string& calculatorGraphConfigFile) {
```

Comment:
I guess abseil comes with Mediapipe and is not a dependency you added, right?

Reply:
Yes, Mediapipe uses it and many of its methods rely on it. If you take a look at the header file, the status messages are handled internally and errors are printed if they occur in debug and false is returned.